.github/workflows/ci.yml

```diff
@@ -11,3 +11,13 @@ jobs:
         uses: golangci/golangci-lint-action@v2
         with:
           version: v1.43.0
+  test:
+    name: Test
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+      - uses: actions/setup-go@v2
+        with:
+          go-version: ^1.17
+      - name: Run tests
+        run: make test
```

Comment:
Will this run the same tests as on CircleCI?

If so, I think this CI step would also need the meroxa machine users credentials to be able to download Go modules correctly

It may look similar to these:
- Github Action with SSH authentication during dependency installation: https://github.com/meroxa/turbine-js/pull/35
- CircleCI workflow with SSH authentication for Go modules: https://github.com/meroxa/cli/pull/290

All of the required credentials are stored in 1Password, but let me know if anything's missing anytime!

Reply:
Nevermind, I had another look at the CI log, but I cannot actually verify what the actual issue is with the tests failing on GH Actions 🤔

Reply:
@jayjayjpg the issue is with the apps init command, which relies on having a GOPATH set up which is where to initialize the go application. This issue could be reproduced locally if you run `make tests` and also having your CLI checked out outside your `GOPATH`.

Reply:
@raulb Got it, thank you for the outline! I'm also curious about the setup you implemented to allow downloads from private repositories correctly -- I'd love to learn more if you happen to have time to pair some time this week or next week!